### PR TITLE
feat: only support celo networks

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

Removes dependency on statsig and just uses the default values. 

### Test plan

- Ensure tokens / activity feed / send / swap flows show the right tokens
- Ensure only Celo shows up in the wallet address page in settings

### Related issues

- Part of ACT-1450

### Backwards compatibility

Y

### Network scalability

N/A

### CODE REVIEW

File: src/statsig/index.ts [hunk 1]
```diff
@@ -89,14 +88,8 @@ function _getDynamicConfigParams<T extends Record<string, StatsigParameter>>({
...
-  const multichainParams = _getDynamicConfigParams(
-    DynamicConfigs[StatsigMultiNetworkDynamicConfig.MULTI_CHAIN_FEATURES]
-  )
-  const filteredParams = {} as { [key: string]: NetworkId[] }
-  Object.entries(multichainParams).forEach(([key, value]) => {
-    filteredParams[key] = value.filter((networkId) => networkId in NetworkId)
-  })
-  return filteredParams
+  // just use defaults, which is celo
+  return DynamicConfigs[StatsigMultiNetworkDynamicConfig.MULTI_CHAIN_FEATURES].defaultValues
```
Comment (satish-ravi) [inline src/statsig/index.ts:93]: this could be cleaned up better, but leaving this as is for now as we might clean up this dyn config altogether in Valora

**Review by jophish — COMMENTED** [review]
Is it possible/easy to just keep the implementation/tests the same, and only update the default values? We could just not have this dynamic config exist in Mento Statsig, so we'd always fall back to the default. That way we could minimize the diff. Or is the issue that we don't yet have a Statsig project set up for Mento?

**Comment by satish-ravi** [comment]
@jophish we could do that too since at the time of launch, I assume we'd have a Statsig account at that point? Is there a ticket for setting up statsig? 
One reason to do it now would be that if we were to demo the app right now, we'd avoid showing all the chains in the wallet address page (since its connected to Valora statsig)

**Comment by jh2oman** [comment]
@satish-ravi @jophish . Fair point on reducing code diff and just waiting till we have a statsig. I assume that we will have a statsig before launch and if the main pain point is that multiple networks are shown in the QR screen then I think it can wait till then and we just do it all in config.

**Comment by satish-ravi** [comment]
Closing this, we can do it when updating statsig credentials

**Pull request closed without merging**